### DISCUSSION TITLE
Best way to analyze TCR repertoire for two different treatments with multiple samples

### DISCUSSION THREAD

**Discussion by gjetro** [milaboratory/mixcr#1509] [Q&A] [answered]
Hi,

I am new to the coding world so I am crawling my way through the MIXCR pipeline. My bulk-RNA seq data consists of one mock (n=4) and one treatment (n=4) I initially used the bulk-RNA seq pipeline for each of my samples which produced output files for each sample (8 in total). I am trying to understand how to compare the TCR repertoire between the treatment group and mock groups, so I attempted to do the overlap post analysis feature. My real question here is what is the best way to use the MixCR pipeline to do this analysis. Should I bulk combine each treatment together at the analyze step (all the mock fastq files then all the treatment fastq files then compare with overlap) or was my thinking correct to normalize the individual samples after the RNA seq pipeline. Thank you for any information.

Tldr: I am trying to compare how TCR reps are different in the treatment group as compared to mock in an infection model with multiple mice per group.

**Comment by mizraelson** [comment]
In general , I would recommend not pooling samples together and compare the groups of 4 samples.

**Reply by gjetro** [reply]
If you individually analyze the samples, then at the end how to you obtain the differences in TCR sequences between each group? I can get the overlap heatmaps, but is there a way in general to obtain the TCR sequences that differ in the treatment group as compared to mock. I cannot find a good representative walkthrough/guide on the homepage.

**Reply by mizraelson** [reply, marked as answer]
So that is a tricky question. The complexity here lies not in the different sequences between two groups, but in a statistically significant change in the abundance of clones found in both conditions. The common approach is to use gene expression algorithms, such as [edgeR](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2796818/), and compare clones as if they were genes. We are currently working on our own pipeline to achieve this more efficiantly and it will be released later this year. 
What you can do is use the [`mixcr exportClonesOverlap`](https://mixcr.com/mixcr/reference/mixcr-exportClonesOverlap/) function to export a table with all clones from all samples. This table will also show you in how many and which exact samples a particular clone was found. From this table, you can easily find clones that differ between groups.

**Reply by gjetro** [reply]
Thanks for your help! I will give that a try and look forward to the pipeline in the future.
